tutorials/automatic-search.md

```
@@ -0,0 +1,146 @@
+# Configurando a Busca Automática
+
+::: tip 🔎 Dica
+Mantenha o mouse pressionado para dar zoom na imagem.
+:::
+
+<style>
+.zoom-img {
+    transition: transform 0.3s;
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)